pkg/container/docker/errors.go

```diff
@@ -0,0 +1,61 @@
+package docker
```

Comment:
These were previously in the client.go file. Moved to a separate file for readability.